syntax = "proto3";
package m2g.pb;

enum AnimationStateType {
  NO_ANIMATION_STATE = 0;
  ANIMATION_STATE_IDLE = 1;
  ANIMATION_STATE_WALKDOWN = 2;
  ANIMATION_STATE_WALKRIGHT = 3;
  ANIMATION_STATE_WALKUP = 4;
  ANIMATION_STATE_WALKLEFT = 5;
}
